scripts/deploy/govcms-pre-deploy-db-update

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+IFS=$'\n\t'
+set -euo pipefail
+
+#
+# Pre-deploy updb
+#
+# This will perform an drush updb on an environment when
+# the environment has the GOVCMS_DEPLOY_PRE_UPDB variable defined.
+#
+# In some use cases databaes updates need to be performed early in
+# the deploy process, even before the site has been bootstrapped
+# (for example with drush status). This provides the deploy agent
+# the ability to perform updates early in the automated flow.
+#
+# The ansible deploy process will add GOVCMS_DEPLOY_PRE_UPDB prior to
```

Comment:
can we please remove this comment as it is confusing for consumer sites - they do not know what "ansible deploy" is.